File: Examples_VI_CtCI_Python/Example5.py
def unordPairs2arr(arrayA, arrayB):
    i = 0
    while (i < len(arrayA)):
        j = 0
        while (j < len(arrayB)):
            k = 0
            while (k < 100000):
                print(str(arrayA[i]) + "," + str(arrayB[j]))
                k += 1
            j += 1
        i += 1

arrayA = [1, 2, 3, 4, 5]
arrayB = [1, 2, 3, 4, 5]
unordPairs2arr(arrayA, arrayB)

File: Examples_VI_CtCI_CPP/Example 5/Example5.cpp
#include <iostream>
// 0 (ab) runtime.

int main() {
	int arrayA[] = {1, 2, 3, 4, 5};
	int arrayB[] = {1, 2, 3, 4, 5};
	int lenA = sizeof(arrayA)/sizeof(arrayA[0]);
	int lenB = sizeof(arrayB)/sizeof(arrayB[0]);
	
	for (int i = 0; i < lenA; i++) {
		for (int j = 0; j < lenB; j++) {
			for (int k = 0; k < 100000; k++) {
				std::cout << arrayA[i] << ", " << arrayB[j] << std::endl;
				}
		}
	}
	return 0;
}
